lib/content-services/src/lib/tag/tag-node-list.component.scss

```diff
@@ -1,7 +1,31 @@
 .adf-tag-node-list {
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    width: inherit;
+
+    &.adf-flex-column {
+        flex-direction: column;
+    }
+
+    .adf-view-more-button {
+        margin: 4px;
+        color: var(--theme-text-color);
+    }
+
+    .adf-full-width {
+        width: 100%;
+    }
+
+    .adf-hidden-btn {
+        color: transparent;
```

Comment:
why not just using "ngIf" to hide the button, with transparent color it might still be visible in the layout

Reply:
This is done on purpose, intially the button is rendered but is "hidden" so I can get its exact width to calculate how many tags can fit into container

Reply:
@MichalKinas @DenysVuika maybe as replacement for transparent we can use visibility:hidden?

Reply:
The problem with visibility changes is the fact that it then shows 0 as width of the button so I can't perform calculations in this case

Reply:
hmm how you apply hidden? Please check here: https://developer.mozilla.org/en-US/docs/Web/CSS/visibility. It still has correct dimensions: 
<img width="1426" alt="image" src="https://user-images.githubusercontent.com/115619721/218489030-a5d828e0-21a7-4e43-9e91-016c2df7ae41.png">

Reply:
I think in the same way as described in the docs, I'll check once again maybe I missed something

Reply:
Ok, I'm asking because there is also angular's [hidden] so in case you used Angular's hidden then you may try pure css visibility: hidden.

Reply:
I've just checked and `visibility: hidden` works as well, must have missed something earlier, @DenysVuika which solution would you prefer?

Reply:
thank you for verifying @MichalKinas :)